src/tests/mockData.js

```diff
@@ -0,0 +1,95 @@
+const collections = [
+  {
+      approvalStatus: "NOT_STARTED",
+      publishComplete: false,
+      isEncrypted: false,
+      collectionOwner: "hello",
+      timeseriesImportFiles: [],
+      id: "anothercollection-91bc818cff240fa546c84b0cc4c3d32f0667de3068832485e254c17655d5b4ad",
+      name: "Another collection",
+      type: "manual",
+      teams: [],
+  },
+  {
+      approvalStatus: "IN_PROGRESS",
+      publishComplete: false,
+      isEncrypted: false,
+      collectionOwner: "PUBLISHING_SUPPORT",
+      timeseriesImportFiles: [],
+      id: "asdasdasd-04917444856fa9ade290b8847dee1f24e7726d71e1a7378c2557d949b6a6968c",
+      name: "asdasdasd",
+      type: "manual",
+      teams: [],
+  },
+  {
+      approvalStatus: "IN_PROGRESS",
+      publishComplete: false,
+      isEncrypted: false,
+      collectionOwner: "PUBLISHING_SUPPORT",
+      timeseriesImportFiles: [],
+      id: "test-collection-12345",
+      name: "Test collection",
+      type: "manual",
+      teams: ["cpi", "cpih"],
+  },
+  {
+      approvalStatus: "ERROR",
+      publishComplete: false,
+      isEncrypted: false,
+      collectionOwner: "PUBLISHING_SUPPORT",
+      timeseriesImportFiles: [],
+      id: "different-collection-12345",
+      name: "Test",
+      type: "manual",
+      teams: ["Team 2"],
+  },
+  {
+      approvalStatus: "COMPLETE",
+      publishComplete: false,
+      isEncrypted: false,
+      collectionOwner: "PUBLISHING_SUPPORT",
+      timeseriesImportFiles: [],
+      id: "test-sau39393uyqha8aw8y3n3",
+      name: "Complete collection",
+      type: "manual",
+      teams: ["Team 2"],
+  },
+];
+
+const items = [
```

Comment:
Wonder if we should split this file out a bit maybe things like
- `/tests/mock/collections`
- `/tests/mock/users` 
etc 

I worry people will just add more and more to this file with just slight adjustments then after a while people will just get into the habit of adding things to the file without checking what is there as it becomes rather massive and we could see duplicates.

Also, wonder if we should change this name `items` to something more meaningful, I had to see how it was used to work out what it was for. Might just be `doubleSelectableBoxControllerItems` or something.

Reply:
I would leave this fir now as it is. I named the collections as per props name they are requested